tools/psa/build_tfm.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
```

Comment:
Could we say just `python`? The script will work for both 2 and 3, right? I would expect on some system, `python` will mean python 3 and on others it'll mean python 2. We shouldn't over specify which version we require here.